app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -179,7 +179,7 @@ def post_refresh_ems_cleanup(_ems, _targets)
 
       def post_process_refresh_classes
         # Return the list of classes that need post processing
-        []
+        [ContainerGroup]
```

Comment:
This should probably be somewhere in the kubernetes refresher

Reply:
What does this do @zeari ?

Reply:
Any classes mentioned here will have self.post_refresh_ems invoked after an inventory refresh

Reply:
Right, I see `amazon/cloud_manager/refresher` and `redhat/infra_manager/refresher` have their own copies